scripts/update_tests.sh

```
@@ -0,0 +1,297 @@
+        if [[ "$next_line" =~ def[[:space:]]+([a-zA-Z0-9_]+)\( ]]; then
+            test_name="${BASH_REMATCH[1]}"
+            SKIP_BACKUP[$test_name]="$line_text"
+        fi
+    done
+}
+
+if ! $check_skip_flag; then
+    echo "Updating Tests"
+
+    if [[ ${#libraries[@]} -eq 0 ]]; then
+        readarray -t libraries <<< $(find ${cpython_path} -type f -printf "%P\n" | grep -vE "$(IFS=\|; echo "${ignored_libraries[*]}")")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)